Dockerfile

```diff
@@ -85,6 +92,24 @@ RUN curl -sL https://deb.nodesource.com/setup_10.x | bash - \
     && apt-get clean \
     && rm -rf /var/lib/apt/lists/*
 
+# Kubernetes dependencies
+RUN curl -fsSL https://download.docker.com/linux/debian/gpg | apt-key add - \
+    && add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/debian stretch stable" \
+    && apt-get update \
+    && apt-get -y install --no-install-recommends docker-ce \
+    && apt-get autoremove -yqq --purge \
+    && apt-get clean && rm -rf /var/lib/apt/lists/*
+
+RUN  curl -Lo kubectl \
+    "https://storage.googleapis.com/kubernetes-release/release/${KUBERNETES_VERSION}/bin/linux/amd64/kubectl" \
+    && chmod +x kubectl \
+    && mv kubectl /usr/local/bin/kubectl
+
+RUN curl -Lo kind \
+    "https://github.com/kubernetes-sigs/kind/releases/download/${KIND_VERSION}/kind-linux-amd64" \
+    && chmod +x kind \
+    && mv kind /usr/local/bin/kind
+
```

Comment:
I'm not sure what you mean? Minicluster is meant to be used with Hadoop. Maybe you refer to Minikube?

Reply:
I think I got it now. You removed the lines in a different PR, right?

Reply:
Yeah. I did not realise that minicluster was a different thing. I moved it indeed in the meantime . I think this will be part of a different PR to use micluster/hadoop etc. images separately from airflow testing. But let's leave it for later.